README.md

```diff
@@ -309,6 +328,15 @@ They will add and remove packages and could leave your OS in a bad
 state.  You should not use them on your personal computer.  Use in a vm
 or container.
 
+Proposed Improvements
```

Comment:
Your proposals are welcome, but they don't belong in the readme file.
Please create issues for these ideas.

Reply:
Okay, I understand. Thank you for the feedback.

Reply:
i agree.  please move these to issues